packages/ui-tree-browser/src/TreeBrowser/TreeCollection/index.tsx

```diff
@@ -100,45 +87,38 @@ class TreeCollection extends Component<TreeBrowserCollectionProps> {
 
   get itemsLevel() {
     const { level, isCollectionFlattened } = this.props
-    // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-    return isCollectionFlattened ? level : level + 1
+    return isCollectionFlattened ? level : level! + 1
```

Comment:
I had to search for the reason why this is always not null. Shall we add a comment here, that the root collection always be level=1 or shall we make it required on the TreeCollection?

Reply:
good catch, as I see it always has a value for us (otherwise it'd be a bug). I'll make it mandatory